views/build/grunt/sass.js

```diff
@@ -8,7 +8,9 @@ module.exports = function(grunt) {
         sass : {
             taoqtitestpreviewer: {
                 files : [
-                    { dest : root + 'js/previewer/provider/item/css/item.css', src :  root + 'js/previewer/provider/item/scss/item.scss'}
+                    { dest : root + 'js/previewer/provider/item/css/item.css', src :  root + 'js/previewer/provider/item/scss/item.scss'},
```

Comment:
you can use a pattern instead, see https://github.com/oat-sa/tao-core/blob/master/views/build/grunt/sass.js#L33

Reply:
Ok, I will check that, thx!